build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -894,6 +906,24 @@ message OutputFile {
   bool is_executable = 4;
 }
 
+// `InlinedOutputFile` is similar to [OutputFile][build.bazel.remote.execution.v2.OutputFile]
+// but has its contents inlined in the message.
+message InlinedOutputFile {
```

Comment:
Why not just add an optional 'contents' field to OutputFile? I'm not a huge fan of the duplication this currently requires, and don't really see why it would be necessary to segregate.

Tangential question: Anyone know why "`OutputFile` is binary-compatible with `FileNode`." ? I assume that was motivated by something in the past, but I can't figure out what. It's probably the wrong time to change that (should be a V3 feature request), but can still be satisfied by reserving the field number in FileNode, as is done with 3 now.

Reply:
+1 to that. I think we should just revert https://github.com/bazelbuild/remote-apis/pull/5 to start with.

I also can't remember for the life of me why OutputFile should be binary compatible with FileNode either. I don't think it should.

Reply:
I created the extra message because of binary compatibility requirement with FileNode. I agree to adding the field to `OutputFile` instead.